plugins/com.google.cloud.tools.eclipse.ui/src/com/google/cloud/tools/eclipse/ui/util/images/SharedImages.java

```diff
@@ -21,11 +21,17 @@
 
 public class SharedImages {
 
+  public static final ImageDescriptor GCP_IMAGE_DESCRIPTOR = AbstractUIPlugin
+      .imageDescriptorFromPlugin("com.google.cloud.tools.eclipse.ui", "icons/obj16/gcp.png"); //$NON-NLS-1$ //$NON-NLS-2$
+
   public static final ImageDescriptor CLOUDSDK_IMAGE_DESCRIPTOR =
-      AbstractUIPlugin.imageDescriptorFromPlugin("com.google.cloud.tools.eclipse.ui",
-          "icons/obj16/cloudsdk.png");
+      AbstractUIPlugin.imageDescriptorFromPlugin("com.google.cloud.tools.eclipse.ui", //$NON-NLS-1$
+          "icons/obj16/cloudsdk.png"); //$NON-NLS-1$
+  public static final ImageDescriptor GCP_WIZARD_IMAGE_DESCRIPTOR = AbstractUIPlugin
+      .imageDescriptorFromPlugin("com.google.cloud.tools.eclipse.ui", "icons/wizban/gcp.png"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
wizban? wizard banner?

Reply:
Yeah, it's the term used in the eclipse plugins